pkgs/servers/search/meilisearch/default.nix

```diff
@@ -18,31 +21,31 @@ rustPlatform.buildRustPackage {
     owner = "meilisearch";
     repo = "MeiliSearch";
     rev = "refs/tags/v${version}";
-    hash = "sha256-D8KAleYaeMv3rCwhGE0IteuDUVk4RiOxsTBe7LhLAKg=";
+    hash = "sha256-OxCxnWAmHMB3JbEG0okJDa1u82DtUCAjXGH3Yq/6ddY=";
   };
 
-  cargoBuildFlags = [
-    "--package=meilisearch"
+  buildInputs = lib.optionals stdenv.isDarwin [
+    Security SystemConfiguration
   ];
 
+  # Default features include mini dashboard which downloads something from the internet.
+  buildNoDefaultFeatures = true;
+
+  # WIP
+  # buildFeatures = lib.optional cudaSupport ["cuda"];
+  # env.CUDA_ROOT = lib.optionalString cudaSupport "${cudaPackages}";
```

Comment:
This will probably be `symlinkJoin { ...; paths = with cudaPackages; [ (forBuild cuda_nvcc) cuda_cudart ... ]; }` where `forBuild = p: p.__spliced.pkgsBuildHost or p`